sdlib/d/gc/tcache.d

```diff
@@ -244,6 +270,22 @@ public:
 	}
 
 private:
+	bool getAppendablePageDescriptor(const void[] slice,
```

Comment:
This is going to be called only by the appendable stuff, so it should be probably be there.